### DISCUSSION TITLE
Allocating static IPs

### DISCUSSION THREAD

**Discussion by cmacrae** [cirruslabs/tart#468] [Ideas]
👋 Thanks so much for sharing Tart with the world! It's _really_ cool!  

I'm building a project with it at the moment. Part of my requirements for this project is that the machine configured and eventually run by Tart have deterministic characteristics. One such characteristic is the IP address.  
Is there any chance we could get a feature to set an IP when we create/run the machine?

**Comment by edigaryev** [comment]
Hello Calum 👋 

You could try using `--net-bridged` and either configure a static address (inside of VM) or configure your LAN's DHCP server to provide IP addresses to Tart VMs in a desirable way (based on a VM MAC-address).
